src/main/java/uk/gov/hmcts/darts/arm/service/impl/ArmApiServiceImpl.java

```diff
@@ -38,9 +38,31 @@ public UpdateMetadataResponse updateMetadata(String externalRecordId, OffsetDate
             .build();
 
         return armApiClient.updateMetadata(
-            String.format("Bearer %s", armTokenResponse.getAccessToken()),
+            getArmBearerToken(),
             armUpdateMetadataRequest
-        ).getBody();
+        );
+    }
+
+    @Override
+    @SneakyThrows
+    public InputStream downloadArmData(String externalRecordId, String externalFileId) {
+        feign.Response response = armApiClient.downloadArmData(
```

Comment:
author note:
I've adopted a solution proposed [here](https://stackoverflow.com/questions/38616808/spring-boot-feignclient-with-image-response-give-error-stream-is-closed)